lgc/patch/Patch.cpp

```diff
@@ -206,7 +191,8 @@ void Patch::addPasses(PipelineState *pipelineState, lgc::PassManager &passMgr, b
 // @param patchTimer : Timer to time patch passes with, nullptr if not timing
 // @param optTimer : Timer to time LLVM optimization passes with, nullptr if not timing
 void LegacyPatch::addPasses(PipelineState *pipelineState, legacy::PassManager &passMgr, ModulePass *replayerPass,
-                            Timer *patchTimer, Timer *optTimer, Pipeline::CheckShaderCacheFunc checkShaderCacheFunc)
+                            Timer *patchTimer, Timer *optTimer, Pipeline::CheckShaderCacheFunc checkShaderCacheFunc,
+                            CodeGenOpt::Level optLevel)
```

Comment:
Could you add the `optLevel` parameter to the comment above ? (And maybe also the other missing parameters ?)